vcluster/_fragments/distro/compat-k0s.mdx

```diff
@@ -1,17 +1,8 @@
-Compatibility matrix showing which host K8s version (left column) is supported by which vCluster k0s distro versions.
+import KubernetesVersions from '@site/vcluster/_partials/deploy/kubernetes-versions.mdx'
 
-|      |   v1.29.1-k0s.0    |   v1.28.2-k0s.0    |   v1.27.6-k0s.0    |   v1.26.9-k0s.0    |
-|------|--------------------|--------------------|--------------------|--------------------|
-| 1.29 | :white_check_mark: | :ok:               | :ok:               | :ok:               |
-| 1.28 | :ok:               | :white_check_mark: | :ok:               | :ok:               |
-| 1.27 | :ok:               | :ok:               | :white_check_mark: | :ok:               |
-| 1.26 | :ok:               | :ok:               | :ok:               | :white_check_mark: |
+### Host cluster compatibility
 
+The Kubernetes distro of vCluster is completely independent of the Kubernetes distro of the host cluster. This means you can run vCluster using k0s distribution regardless of the distribution type of the host cluster.
```

Comment:
Just a thought, feel free to ignore if its too tedious, but this verbiage could be put into a component that takes a distro attribute so it would be somethings like `<DistroAdmonition distro="k0s" />` in all the places this is used.  I don't have better names at the moment to not have "distro" in the component and attr name.  _If_ you do go this route, it would be nice to have the links to the distro pages like they do under the `Kubernetes Distros for vCluster` list.